client/stylesheets/_modal.scss

```diff
@@ -371,6 +371,16 @@
 	margin: 8px 0 15px 0 !important;
 }
 
+.profileReceivedPriceOriginal {
+	font-size: 0.8rem;
+	vertical-align: middle;
+	margin-left: 6px;
+}
+
+.profileReceivedPrice b, .profileReceivedPrice span {
+	vertical-align: middle !important; 
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Line contains trailing whitespace